client/knowledge_base_search/src/containers/SearchContainer/SearchContainer.js

```diff
@@ -49,50 +53,38 @@ class SearchContainer extends Component {
 
   render() {
     return (
-      <section className="_full-width-row search_container--section">
-        <div className="_container _container_large">
+      <section className='_full-width-row search_container--section'>
+        <div className='_container _container_large'>
           <form onSubmit={this.handleOnSubmit}>
             <Tabs selected={0}>
-              <Pane label="Preset questions">
-              <div className='clickable-tabs'>
-                {
-                  this.state.random_queries.map((query, index) => {
-                    return (
-                      <span key={'questionButton-'+index}>
-                        <button className={query === this.state.search_input
-                          ? 'active_button clickable-tab'
-                          : 'clickable-tab'}
-                          type='button'
-                          disabled={this.props.isFetching}
-                          onClick={() => {
-                            this.handleQuestionClick(query, index);
-                          }}>
-                            {query}
-                        </button>
-                      </span>
-                    )
-                  })
-                }
-              </div>
+              <Pane label='Preset questions'>
+                <QuestionBarContainer
+                  currentQuery={this.state.search_input}
+                  onQuestionClick={this.handleOnQuestionClick}
+                  presetQueries={this.state.presetQueries}
+                  isFetching={this.props.isFetching}
+                />
               </Pane>
               <Pane label='Custom question'>
-                <span className="positioned--icon">
-                  <Icon type="search" />
-                </span>
-                <TextInput
-                  id="search_input"
-                  placeholder="Enter words, phrase, or a question about travel"
-                  value={this.state.search_input}
-                  onInput={this.handleOnInput}
-                  style={{width: '100%'}}
-                  disabled={this.props.isFetching}
-                />
-                <button
-                  className="white--button"
-                  disabled={this.props.isFetching}
-                >
-                  Retrieve Answers
-                </button>
+                <div className='custom_question--div'>
+                  <span className='positioned--icon'>
+                    <Icon type='search' />
+                  </span>
+                  <TextInput
+                    id='search_input'
+                    placeholder='Enter words, phrase, or a question about travel'
+                    value={this.state.search_input}
+                    onInput={this.handleOnInput}
+                    style={{width: 'calc(100% - 3rem)'}}
```

Comment:
boooo. Why not add a class here? If the `TextInput` components doesn't support a `className` prop, I still think it's better to select it from a wrapping class and apply this in the css file.